src/content/tutorials-landing.json

```
@@ -1,147 +1,163 @@
 				},
 				{
 					"href": "/consul/tutorials/network-infrastructure-automation",
-					"text": "Network Automation with CTS"
+					"text": "Network automation"
 				}
 			],
 			"featuredCollectionSlugs": [
 				"consul/get-started-hcp",
 				"consul/get-started-kubernetes",
 				"consul/get-started-vms"
+			],
+			"certificationProgram": {
+				"slug": "networking-automation",
+				"title": "Get certified",
+				"description": "Validate your networking automation skills by studying for and passing the Consul Associate certification exam."
+			}
+		},
+		"nomad": {
+			"featuredUseCases": [
+				{
+					"href": "/nomad/tutorials/autoscaler",
+					"text": "Auto-scaled applications"
+				},
+				{
+					"href": "/nomad/tutorials/stateful-workloads",
+					"text": "Stateful workloads"
+				},
+				{
+					"href": "/nomad/tutorials/task-deps",
+					"text": "Task dependencies"
+				}
+			],
+			"featuredCollectionSlugs": [
+				"nomad/get-started",
+				"nomad/cluster-setup",
+				"nomad/manage-jobs"
 			]
 		},
-		"vault": {
-			"certificationProgram": {
-				"slug": "security-automation",
-				"title": "Verify your Security Automation skills",
-				"description": "Ready to demonstrate your advanced Vault Enterprise skills in production? Successfully complete a lab-based exam and earn the Vault Operations Professional certification."
-			},
+		"packer": {
 			"featuredUseCases": [
 				{
-					"href": "/vault/tutorials/adp",
-					"text": "Advanced Data Protection"
+					"href": "/packer/tutorials/cloud-production/golden-image-with-hcp-packer",
+					"text": "Golden image pipelines"
 				},
 				{
-					"href": "/vault/tutorials/db-credentials",
-					"text": "Database Credential Rotation"
+					"href": "/packer/tutorials/cloud-production/multicloud",
+					"text": "Standardized images across clouds"
 				},
 				{
-					"href": "/vault/tutorials/encryption-as-a-service",
-					"text": "Data Encryption"
+					"href": "/packer/tutorials/cloud-production/github-actions",
+					"text": "Automated image building"
 				}
 			],
 			"featuredCollectionSlugs": [
-				"vault/getting-started",
-				"vault/getting-started-ui",
-				"vault/cloud"
+				"packer/docker-get-started",
+				"packer/aws-get-started",
+				"packer/hcp-get-started"
 			]
 		},
 		"boundary": {
 			"featuredUseCases": [
 				{
 					"href": "/boundary/tutorials/hcp-administration",
-					"text": "HCP Boundary Resource Management Workflows"
+					"text": "Resource management"
 				},
 				{
 					"href": "/boundary/tutorials/oss-administration",
-					"text": "Self-Hosted Boundary Resource Management Workflows"
+					"text": "Self-hosted resource management workflows"
 				},
 				{
 					"href": "/boundary/tutorials/access-management",
-					"text": "Secure Access Management Workflows"
+					"text": "Secure access management workflows"
 				}
 			],
 			"featuredCollectionSlugs": [
 				"boundary/hcp-getting-started",
 				"boundary/oss-getting-started"
 			]
```

Adding the 3rd collection:

```suggestion
			"featuredCollectionSlugs": [
				"boundary/hcp-getting-started",
				"boundary/oss-getting-started",
				"boundary/credential-management"
			]
```